public/js/app.js

```diff
@@ -49506,8 +49506,8 @@ __webpack_require__.r(__webpack_exports__);
 /*! no static exports found */
 /***/ (function(module, exports, __webpack_require__) {
 
-__webpack_require__(/*! C:\Users\dell01\Desktop\InterConnWeb\resources\js\app.js */"./resources/js/app.js");
-module.exports = __webpack_require__(/*! C:\Users\dell01\Desktop\InterConnWeb\resources\sass\app.scss */"./resources/sass/app.scss");
+__webpack_require__(/*! C:\Users\Greatest\Desktop\code\interconn\navbar\interconnweb\resources\js\app.js */"./resources/js/app.js");
+module.exports = __webpack_require__(/*! C:\Users\Greatest\Desktop\code\interconn\navbar\interconnweb\resources\sass\app.scss */"./resources/sass/app.scss");
```

Comment:
I am wondering why we are going to be using `Greatest` keywords on our platform

Reply:
@reifred aint you looking at my default URL just like the previous and that comes from the machine being used, and the last I checked I can name my Machine anything I want boss because the previous shows your dell01, or was that @NicholusMuwonge

Reply:
Good to know